common/services/reference-data.test.js

```diff
@@ -70,895 +70,829 @@ const mockRegions = [
   },
 ]
 
+const locationInclude = ['suppliers']
+const regionInclude = ['locations']
+
 describe('Reference Data Service', function () {
-  context('', function () {
```

Comment:
A large amount of this diff is because of the removal of the empty `context` block.

It wasn't being used so this changes the indentation for all the tests.